python/README.md.in

```diff
@@ -65,6 +65,9 @@ source $CONDA_PREFIX/lib/python3.11/site-packages/distributed_interfaces/activat
 
 [//]: # (End conda install)
 
+**Warning (conda-forge)**: Installing `cudaq` from the `conda-forge` channel on recent versions of Python can lead to segmentation faults. 
+If you see such errors, please switch to the old Python version.
```

Comment:
I find "the old Python version" part not clear. Is it possible to specify the latest working versions (3.10.15, 3.11.10 and 3.12.7) or to insert a link to either #2999 or #3104?